saritasa_invocations/pip.py

```diff
@@ -15,7 +15,7 @@ def install_dependencies(
 
 
 @invoke.task
-def compile_dependencies(
+def compile(
```

Comment:
This looks like a breaking change, could you search for a way to provide "backward compatibility"?

Reply:
What about this? Code also updated

<img width="990" alt="Снимок экрана 2024-02-29 в 10 12 52" src="https://github.com/saritasa-nest/saritasa-invocations/assets/30920379/27198b20-5836-46ea-80dc-fabc70010973">

Reply:
Looks good. It would be nice if this command will be hidden from `inv -l` and ZSH autocompletion

Reply:
As I can see invoke hasn't `hide` option for tasks https://github.com/pyinvoke/invoke/pull/723

But I found `aliases` option :) It brings less code. See my second [commit](https://github.com/saritasa-nest/saritasa-invocations/pull/75/commits/49a1126152350466dd9613ce215615bc236eff0a) 

<img width="724" alt="image" src="https://github.com/saritasa-nest/saritasa-invocations/assets/30920379/a79f97ce-437a-4ffe-9943-7465d025de07">

Reply:
Actually, I think we should add alias `pip.compile` to existing `pip.compile_dependencies` function because there may be usages of `compile_dependencies` **function** from other tasks